README.md

```diff
@@ -0,0 +1,65 @@
+
+# Project Name
+
+> this project helps you keep track of reserved missions and rockets on ypor profile add or remove the on your profile.
+
+
+## Built With
+
+- react
+- bootstraps
+- redux
+
+**To get this project locally**
+
+## Run the following command on your terminal
+
+
+```
+git clone https://github.com/businesskaniki/spacex.git
+
+```
+```
+cd spacex
+
+```
+```
+npm install
+
+```
+```
+npm start
+
+```
```

Comment:
- [ ] OPTIONAL
It will be good idea to also add instructions on how to run unit test cases for this project.